Index: screenruler-0.960+bzr41/screenruler.rb
===================================================================
--- screenruler-0.960+bzr41.orig/screenruler.rb	2014-01-16 11:04:00.063701352 +0100
+++ screenruler-0.960+bzr41/screenruler.rb	2014-01-16 11:04:47.919408300 +0100
@@ -19,7 +19,7 @@
  #  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  ###############################################################################
 
-Dir.chdir(File.dirname(File.expand_path(File.symlink?(__FILE__) ? File.readlink(__FILE__) : __FILE__)))		# So that this file can be run from anywhere
+Dir.chdir('/usr/share/screenruler')
 $LOAD_PATH << './utils'
 
 require 'gettext'		# Internationalization Support

