Index: screenruler/ruler_window.rb
===================================================================
--- screenruler.orig/ruler_window.rb
+++ screenruler/ruler_window.rb
@@ -243,7 +243,7 @@ private
 
 		# Loop, drawing ticks (top and bottom) and labels
 		repetitions, tick_index = 0, 0
-		loop(pixels_per_tick, length + OVERDRAW, pixels_per_tick) { |x|
+		sr_loop(pixels_per_tick, length + OVERDRAW, pixels_per_tick) { |x|
 			x = x.floor + 0.5		# Cairo likes lines in the 'center' of pixels
 
 			tick_size = @@tick_sizes[ unit.tick_pattern[tick_index, 1].to_s ]
@@ -332,7 +332,7 @@ private
 
 		# Fill with 'horizontal' lines
 		cr.set_source_color($preferences_window.foreground_color)
-		loop(@menu_box.y + 2.5, @menu_box.y + @menu_box.height + -1.5, 2) { |y|
+		sr_loop(@menu_box.y + 2.5, @menu_box.y + @menu_box.height + -1.5, 2) { |y|
 			cr.move_to(@menu_box.x + 2.0, y)
 			cr.line_to(@menu_box.x + @menu_box.width - 1, y)
 		}
Index: screenruler/utils/addons_ruby.rb
===================================================================
--- screenruler.orig/utils/addons_ruby.rb
+++ screenruler/utils/addons_ruby.rb
@@ -59,7 +59,7 @@ module Kernel
 		list.each { |file| orig_require(file) }
 	end
 
-	def loop(from, to, step=1)
+	def sr_loop(from, to, step=1)
 		i = from
 		while i <= to
 			yield i
